docker/Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG PROFILE
 # https://hub.docker.com/_/rust
 # our host is based on bullseye/sid => rust:latest
 # ==========================
-FROM rust:latest as builder
+FROM rust:1.64 as builder
```

Comment:
It's not a good idea to use older rust toolchain (base image), I think `rust:bullseye` is enough, see `Image Variants` in https://hub.docker.com/_/rust

Let me change it directly :)